sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -93,6 +93,10 @@ def plot(self, *, include_values=True, cmap='viridis',
             Format specification for values in confusion matrix. If `None`,
             the format specification is 'd' or '.2g' whichever is shorter.
 
+        values_size : int, default=None
+            Font size of the values in the confusion matrix. If `None`,
+            `plt.rcParams["font-size"]` will be used.
```

Comment:
I think it's better to be generic here with `text_kw`, which is passed into `ax.text`.

Note this is very similar to `im_kw` in https://github.com/scikit-learn/scikit-learn/pull/20753